Habitica/src/com/magicmicky/habitrpgwrapper/lib/models/tasks/Task.java

```diff
@@ -112,8 +119,8 @@ public void setId(String id) {
     /**
      * @return the notes
      */
-    public String getNotes() {
-        return notes;
+    public CharSequence getNotes() {
+        return markdownParser.parseMarkdown(notes);
```

Comment:
like in the first version, the parsed markdown shouldn't be inside `Task.java`.  Could you please move it to the ViewHolders / Activity / Fragments if needed